code/lambda_handlers.py

```diff
@@ -0,0 +1,190 @@
+from __future__ import print_function
+import sys
+import log_helper
+sys.path.append('lib')
+import numpy
+from jsonschema import validate
+import validation_json_schemas as schemas
+
+logger = log_helper.getLogger(__name__)
+
+
+def fv_handler(request, context):
+    """
+    Future Value calculation
+    :param request: Dict containing the parameters to pass to the formula.
+    :param context: Lambda execution context
+    :return: Dict with a 'result' entry containing the result of the calculation
+    """
+    schema = schemas.fv_schema
+
+    try:
+        validate(request, schema)
+        logger.info("FV request: {}".format(request))
+        args = [request['rate'], request['nper'], request['pmt'], request.get('pv', 0), request.get('type', 0)]
+        logger.info("Calling numpy.fv with args: {}".format(args))
+        return {'result': numpy.fv(*args)}
```

Comment:
nit: We can probably add a private helper to make this more DRY:

```
return __call_numpy('fv', args)
```

Then move the log message and result object creation into the helper.